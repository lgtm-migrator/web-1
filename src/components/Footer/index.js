import React from "react"
import { Link as GatsbyLink } from "gatsby"

import Box from "@material-ui/core/Box"
import Link from "@material-ui/core/Link"
import Typography from "@material-ui/core/Typography"

export default ({ siteTitle }) => {
  return (
    <Box component="footer" display="block">
      <Typography paragraph={true}>
        Â© {new Date().getFullYear()}{" "}
        <Link component={GatsbyLink} to="/">
          {siteTitle}
        </Link>
      </Typography>
    </Box>
  )
}
