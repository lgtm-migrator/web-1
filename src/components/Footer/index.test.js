import React from "react"
import renderer from "react-test-renderer"
import Footer from "."

describe("Footer", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Footer siteTitle="Test site title" />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
