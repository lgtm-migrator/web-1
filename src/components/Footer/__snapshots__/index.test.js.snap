// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<footer
  className="MuiBox-root MuiBox-root-2"
  itemScope={true}
  itemType="https://schema.org/WPFooter"
  role="contentinfo"
>
  <div
    className="MuiContainer-root MuiContainer-maxWidthSm"
  >
    <ul
      className="MuiList-root makeStyles-root-11 MuiBox-root MuiBox-root-12 makeStyles-root-11"
    >
      <li
        className="MuiListItem-root MuiListItem-gutters"
        disabled={false}
      >
        <p
          className="MuiTypography-root MuiTypography-body1 MuiTypography-paragraph"
        >
          <small>
            © 
            2020
             
            <strong>
              Test site title
            </strong>
          </small>
        </p>
      </li>
      <li
        className="MuiListItem-root MuiListItem-gutters"
        disabled={false}
      >
        <a
          className="MuiTypography-root MuiLink-root MuiLink-underlineAlways MuiTypography-colorInherit"
          href="/ajuda"
          onBlur={[Function]}
          onFocus={[Function]}
        >
          Ajuda
        </a>
      </li>
      <li
        className="MuiListItem-root MuiListItem-gutters"
        disabled={false}
      >
        <a
          className="MuiTypography-root MuiLink-root MuiLink-underlineAlways MuiTypography-colorInherit"
          href="/privacidade"
          onBlur={[Function]}
          onFocus={[Function]}
        >
          Privacidade
        </a>
      </li>
      <li
        className="MuiListItem-root MuiListItem-gutters"
        disabled={false}
      >
        <a
          className="MuiTypography-root MuiLink-root MuiLink-underlineAlways MuiTypography-colorInherit"
          href="/termos"
          onBlur={[Function]}
          onFocus={[Function]}
        >
          Termos de uso
        </a>
      </li>
    </ul>
  </div>
</footer>
`;
